### PR TITLE
Vanilla installer instructions 

### PR DESCRIPTION
**Pull request by Madis0**
https://github.com/Fabulously-Optimized/fabulously-optimized/issues/627

### CODE REVIEW

File: install-instructions.md [hunk 1]
```diff
@@ -103,27 +101,28 @@ P.S. If you get Deck-specific issues, try asking on [r/SteamDeck](https://www.re
...
+1. Open the [releases page](https://github.com/Fabulously-Optimized/vanilla-installer/releases/latest) and download the file with the `.exe` extension. (Example: `Vanilla Installer-GUI v0.1.0.exe`)
+2. If it asks you where to save your file, select your Downloads folder (`%USERPROFILE%\Downloads`)
+3. Navigate to the Downloads folder and double-click the file you just downloaded.
+   * If you see a Microsoft Defender SmartScreen prompt, click `More info`, then `Run Anyway`.
+4. Select the Minecraft version and directory, and click `Install`.
+5. You're all done!
+
+#### macOS
+
+TODO
```
Comment (Madis0) [inline install-instructions.md:115]: TODO

File: install-instructions.md [hunk 2]
```diff
@@ -92,29 +92,28 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
 
 ### [Minecraft Launcher](https://www.minecraft.net/en-us/download) (vanilla)
 
-For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Installer.
```
Comment (Madis0) [inline install-instructions.md:95]: Bring that part back through mrpack-to-zip tooö

File: install-instructions.md [hunk 3]
```diff
@@ -102,7 +102,12 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
...
+1. Download [vanilla installer](https://download.fo/vanilla) and run it
```
Comment (Madis0) [inline install-instructions.md:105]: todo: windows, macos, linux quirks that will appear on running it

File: install-instructions.md [hunk 4]
```diff
@@ -102,7 +102,15 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
...
+    * Windows:
+    * macOS:
+    * Linux:
```
Comment (Imzxhir) [inline install-instructions.md:108]:
```suggestion
    * macOS: Open the .dmg file, double click the 'Fabulously Optimized Installer' and click 'Open' when asked in a prompt.
```

**Review by Imzxhir — CHANGES_REQUESTED** [review]

File: install-instructions.md [hunk 5]
```diff
@@ -102,7 +102,19 @@ You need [Java 17 or higher](https://adoptium.net/) to play the game.
...
+The Vanilla Installer is currently in open beta. If you run into any issues, please report them on Discord and use the manual method.
+
+1. Download [vanilla installer](https://download.fo/vanilla) and run it
+    * Windows: Open the .exe file, click "More info" and then "Run anyway".
+    * macOS: Open the .dmg file, **right click** the "Fabulously Optimized Installer" and click "Open", click "Open" again when asked in a prompt.
+    * Linux: Open the .appimage file and click "Run once" when prompted.
+2. Select preferred FO/Minecraft version and install
+3. Run Minecraft Launcher. You'll see the new installation active, click PLAY.
+
+Note: you may now see a new file _paigaldaja_meta.json_ next to the installer. This is a configuration file that ensures smoother upgrades and remembers some of your install settings.
```
Comment (vgskye) [inline install-instructions.md:114]: this is technically incorrect here; `paigaldaja_meta.json` is saved in the profile dir

**Pull request merged**